www-client/links/links-2.20.recipe

```diff
@@ -4,10 +4,10 @@ HOMEPAGE="http://links.twibright.com/"
 COPYRIGHT="1999-2019 Mikulas Patocka
 	2000-2011 Petr Kulhavy, Karel Kulhavy, Martin Pergel"
 LICENSE="GNU GPL v2"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.gz"
-CHECKSUM_SHA256="37299f804f22c945bf63e22a7bc4418bbb8144b410c0ced70b82ebe6f9e3c82b"
-PATCHES="links-$portVersion.patchset"
+CHECKSUM_SHA256="900c9e461db09757f7d856086925d5be4087a1f7171db5194f38f64e0987035c"
+#PATCHES="links-$portVersion.patchset"
```

Comment:
Was the entire patchset really merged?

Reply:
Looks like they were upstreamed.

Reply:
Fixed for now.

Reply:
All legacy patches reviewed and confirmed in v2.20.1. Feel free to fully test for any further feature add.

Reply:
It's been merged, and even improved upon by links author.